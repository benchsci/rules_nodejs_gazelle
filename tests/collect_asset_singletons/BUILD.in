# gazelle:js_root
# gazelle:js_web_asset json,css,scss
# gazelle:js_collect_all_assets
