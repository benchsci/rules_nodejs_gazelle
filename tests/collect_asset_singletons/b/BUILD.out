load("@com_github_benchsci_rules_nodejs_gazelle//:defs.bzl", "web_assets")

web_assets(
    name = "style_css",
    srcs = ["style.css"],
)
